### PR TITLE
Test sinatra base gem

### PR DESCRIPTION
**Pull request by MarinaMuttik**
As currently, error:
> cannot load such file -- rack/showexceptions (LoadError)

(as in Gemfile comment), if the sinatra-base gem is specified to ~>1.4 it will run, however with a warning:
> warning: constant ::Fixnum is deprecated

When as in master it is just the sinatra gem and therefore using a newer version of rack etc (2.0.5) this works fine.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,5 +2,6 @@ source 'https://rubygems.org'
 
 ruby "2.5.1"
 
-gem "sinatra"
+gem "sinatra-base"
```
Comment (dansteele) [inline Gemfile:5]: Why did you change it to `sinatra-base`?

Comment (MarinaMuttik) [reply Gemfile:5]: bc that was what wasn't working but I don't really *need* the full sinatra gem I believe

Comment (dansteele) [reply Gemfile:5]: Whilst that might be true, I would leave in the whole gem for now :) Whilst definitely a good idea to not bring in more of a dependency than you need, I don't think it's much of a learning prio for now 🙈
